src/Files.App/Helpers/UI/UIFilesystemHelpers.cs

```diff
@@ -26,6 +26,26 @@ public static async Task PasteItemAsync(string destinationPath, IShellPage assoc
 			}
 		}
 
+		public static async Task PasteItemAsShortcutAsync(string destinationPath, IShellPage associatedInstance)
```

Comment:
Can you move this to the action itself?

Reply:
It can, but we might be better off keeping it in the helper. There is a lot of duplicate code in this helper that can be reduced.

Reply:
I see. FWIW, this class will be retired in advance of storage abstraction's helpers:

```cs
StorageHelpers.CopyAsShortcutAsync(/*IModifiableStorable*/ storable);
```

Reply:
Lets keep in the helper, and we can remove when we switch to the storage abstractions.

Reply:
Absolutely right.